tests/nbt_timing_benchmark.py

```diff
@@ -0,0 +1,39 @@
+from itertools import repeat
```

Comment:
If this file is only benchmark that we should manually run, it should probably be in a new "benchmarks" folder rather than in "tests".
If it's in tests, we can add a "limit" (for example, 2 seconds) that if it's slower than that, it fails (which would make it a test).
Also, I'm not sure if it tests Amulet itself or the NBT repo. If it's for the NBT repo, we might want to move the benchmark there.

Reply:
I think it would be better suited in the NBT repo, I'll remove it from here and we can add a better comparison test in the other repo at a later date